backend/src/main/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtils.java

```diff
@@ -448,12 +449,30 @@ public static List<FeedbackMessage> validateDateFormat(ValueOrError input) {
   }
 
   public static List<FeedbackMessage> validateDateTime(ValueOrError input) {
-    List<FeedbackMessage> errors = new ArrayList<>(validateRegex(input, DATE_TIME_REGEX));
-    if (input.getValue() != null
-        && errors.isEmpty()
-        && input.getValue().matches(TIMEZONE_SUFFIX_REGEX)) {
-      errors.addAll(validateDateTimeZoneCode(input));
+    String value = parseString(input.getValue());
+    List<FeedbackMessage> errors = new ArrayList<>();
+
+    if (value == null) {
+      return errors;
     }
+
+    try {
+      convertToZonedDateTime(value);
+
+      if (hasTimezoneSubstring(value)) {
+        errors.addAll(validateDateTimeZoneCode(input));
+      }
+    } catch (DateTimeParseException | StringIndexOutOfBoundsException e) {
+      errors.add(
+          FeedbackMessage.builder()
+              .scope(ITEM_SCOPE)
+              .fieldHeader(input.getHeader())
```

Comment:
```suggestion
              .fieldHeader(input.getHeader())
              .source(ResultUploadErrorSource.SIMPLE_REPORT)
```

I think we want the source set to SR to indicate that this error is coming from our validation step in the [Result Upload Error](https://www.simplereport.gov/metabase/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7ImRhdGFiYXNlIjoyLCJ0eXBlIjoicXVlcnkiLCJxdWVyeSI6eyJzb3VyY2UtdGFibGUiOjUxfX0sImRpc3BsYXkiOiJ0YWJsZSIsInZpc3VhbGl6YXRpb25fc2V0dGluZ3MiOnt9fQ==) table

sorry if we're setting it somewhere else and I missed it

Reply:
Thanks @mehansen ; I've committed this suggestion.